src/main/java/net/minestom/server/coordinate/Point.java

```diff
@@ -261,12 +261,12 @@ default boolean sameBlock(int blockX, int blockY, int blockZ) {
     }
 
     /**
-     * Gets if two points are in the same chunk.
+     * Gets if two points are in the same block.
```

Comment:
`Checks` instead of `Gets`

Reply:
I wanted to stay consistent with other methods, but yes, this is better, I guess we can change all of them.